doc.go

```diff
@@ -198,6 +198,8 @@ Users can use SetLogger in driver.go to set a customized logger for gosnowflake
 In order to enable debug logging for the driver, user could use SetLogLevel("debug") in SFLogger interface
 as shown in demo code at cmd/logger.go. To redirect the logs SFlogger.SetOutput method could do the work.
 
+# If you want to define S3 client logging, override S3LoggingMode variable using configuration: https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#ClientLogMode
```

Comment:
Is there a plan for similar in Azure and Google
e.g. https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore/log

Reply:
When I have some time, yes :)